rest-nvd/docker-compose.yml

```diff
@@ -3,6 +3,8 @@ version: '2'
 services:
   rest-nvd:
     build: .
+    container_name: vulnerability-assessment-tool-rest-nvd
```

Comment:
Hi @sumeetpatil is this `container_name` used somewhere? if not, we could drop it, for consistency with the other service definitions.

Reply:
other services have container names - https://github.com/eclipse/steady/blob/master/docker/docker-compose.yml. 
maybe I can drop it for now in rest-nvd/docker-compose.yml and add it in the main compose file?

these container-names are then later used in pushing images to docker hub, checking the number of containers running and more. 
I will check what can be done here and create a new PR for fixing the issue with the container names.

As per the docker official site, this is what they say - 
Docker container names must be unique, you cannot scale a service beyond 1 container if you have specified a custom name. Attempting to do so results in an error.
https://docs.docker.com/compose/compose-file/#container_name

Reply:
My bad, indeed we have container names everywhere, in our current setup; please, ignore my comment.

Reply:
Hi @copernico ,
I take my words back. I am checking the usage of container names once again. I also have to checkin the code for docker-compose.build.yml file